src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin: 0;
+    cursor: pointer;
+  }
+
+  &__star:not(:first-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars__star:hover ~ div,
+  .stars__star:hover {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Cool, but use static state, and set 4 stars active in this card but also you should have modifiers for all other amount of stars from 0 to 5